tracer/test/Datadog.Trace.ClrProfiler.IntegrationTests/WebRequestTests.cs

```diff
@@ -85,8 +86,10 @@ private void RunTest(string metadataSchemaVersion)
             using (var agent = EnvironmentHelper.GetMockAgent())
             using (ProcessResult processResult = RunSampleAndWaitForExit(agent, arguments: $"Port={httpPort}"))
             {
-                agent.SpanFilters.Add(s => s.Type == SpanTypes.Http);
-                var spans = agent.WaitForSpans(expectedSpanCount).OrderBy(s => s.Start);
+                var allSpans = agent.WaitForSpans(expectedAllSpansCount).OrderBy(s => s.Start);
+                allSpans.Should().OnlyHaveUniqueItems(s => new { s.SpanId, s.TraceId });
```

Comment:
Before the product change (and after we started collecting all spans from the application), we would see this error message from the test:

```
Expected allSpans to only have unique items on new <>f__AnonymousType11`2(SpanId == s.SpanId, TraceId == s.TraceId), but items {TraceId: 3286320956883792616, SpanId: 7849642159726264990, Name: GetResponseWithDistributedTracingHeaders, Resource: GetResponseWithDistributedTracingHeaders, Service: Samples.WebRequest, TraceId: 3286320956883792616, SpanId: 7849642159726264990, Name: http.client.request, Resource: GET localhost:49189/?/, Service: Samples.WebRequest, TraceId: 3286320956883792616, SpanId: 12757857473955457543, Name: GetResponseAsyncWithDistributedTracingHeaders, Resource: GetResponseAsyncWithDistributedTracingHeaders, Service: Samples.WebRequest, TraceId: 3286320956883792616, SpanId: 12757857473955457543, Name: http.client.request, Resource: GET localhost:49189/?/, Service: Samples.WebRequest} are not unique.
```